templates/form.html

```diff
@@ -18,20 +18,22 @@
                     <span id="error-message2"> </span>
                 </div>
             </div>
-	    <div class="row" >
+	    <div class="row">
                 <div class="col-md-6 mb-3">
                     <label for="startAirport" class="form-label">Start Airport</label>
-                    <input type="text" class="form-control" id="startAirport" name="startAirport" placeholder="LAX" autocomplete="off">
+                    <input type="text" class="form-control" id="startAirport" name="startAirport" placeholder="LAX" autocomplete="off" oninput="updateAirportOptions('startAirport', 'startAirportSelect')" onclick="updateAirportOptions('startAirport', 'startAirportSelect')">
+		                <select class="form-select mt-2" id="startAirportSelect" name="startAirport" onclick="selectAirport('startAirport', 'startAirportSelect', event)"></select>
```

Comment:
It seems that when clicking the dropdown, it takes a second to load because it is getting all the possible airports. Is that correct? Whatever the case, is there a way to shorten the list so that it only shows a limited number of airports and show more when scrolling (so it extends the list when scrolling).